docs/reference/commandline/update.md

```diff
@@ -21,12 +21,13 @@ Usage:  docker update [OPTIONS] CONTAINER [CONTAINER...]
 Update configuration of one or more containers
 
 Options:
-      --blkio-weight value          Block IO (relative weight), between 10 and 1000
+      --blkio-weight uint16         Block IO (relative weight), between 10 and 1000, or 0 to disable (default 0)
```

Comment:
Oh, can we make it so that this just outputs `int`? We already describe "between 10 and 1000", so using a rather technical `uint16` doesn't add much :smile: